nx.json

```diff
@@ -0,0 +1,71 @@
+{
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "nx/tasks-runners/default",
+      "options": {
+        "cacheableOperations": [
+          "build",
+          "build:babel",
```

Comment:
Interesting - so the cacheable actions are the package actions, not the top level actions?

Reply:
Yep. This was generated from their CLI tool which generates a list of operations and asks which are cacheable